src/v2/Components/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx

```diff
@@ -43,19 +43,18 @@ const ArtistItem: React.FC<
   const toggleArtistSelection = (selected, slug) => {
     let updatedValues = artistIDs
 
+    // Move followed artists to the explicit `artistIDs` list
+    if (isFollowedArtistCheckboxSelected) {
+      updatedValues = [...artistIDs, ...followedArtistSlugs]
+    }
```

Comment:
**question**: maybe we need to move to `artistIDs` list only the ids of those artists that are displayed in Artists filters?

#### Example
* `followedArtistSlugs` contains following values `["Artist #1", "Artist #2", ..., "Artist #100"]`
* In `Artists` filter only `Artist #1` and `Artist #2` from `followedArtistSlugs` are displayed
* When any artist option item has been selected/unselected, we automatically unselect `Artists I follow` and move ids to `artistIDs`
* in this case, should we move to `artistIDs` all values from `followedArtistSlugs` or only the IDs of those artists that are displayed in the filter?